pyston/nitrous/Execution.cpp

```diff
@@ -1175,6 +1176,16 @@ void Interpreter::visitCallSite(CallSite CS) {
     case Intrinsic::dbg_value:
       return;
 
+    case Intrinsic::frameaddress: {
```

Comment:
I think we should be able to remove this again?!?
I worry a bit that w could run into some code which is using this in the future and we silently do something really strange while tracing it instead of obvious error code.